crates/viewer/re_view_spatial/src/eye.rs

```diff
@@ -276,6 +287,27 @@ impl ViewEye {
         }
     }
 
+    /// Compute the actual speed depending on the [`EyeMode`].
+    fn compute_speed_for_mode(&self, bounding_boxes: &SceneBoundingBoxes) -> f32 {
```

Comment:
going with naming scheme we have for blueprint properties (and this should be one eventually) this would be `fallback_speed_for_mode` since "fallback" is the term we use for the dynamically calculated values we use when a blueprint property wasn't specified

Reply:
Renamed to `fallback_speed_for_mode()`